packages/core/src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -189,19 +189,22 @@ export class DdRumResourceTracking {
     /**
      * Starts tracking resources and sends a RUM Resource event every time a network request is detected.
      */
-    static startTracking(tracingSamplingRate: number): void {
-        DdRumResourceTracking.startTrackingInternal(
-            XMLHttpRequest,
+    static startTracking({
```

Comment:
nice trick, but just for our understanding can you please explain a bit how a previous `startTracking(tracingSamplingRate)` would translate to this new method definition ? For example I see in the method bellow that you are calling the `internal` method :
`DdRumResourceTracking.startTrackingInternal(XMLHttpRequest, { tracingSamplingRate })`
but I am wondering how this will translate if I am calling the static method directly with an object:
`DdRumresourceTracking.startTracking({trackingSamplingRate:10})`

Reply:
In js, `{ tracingSamplingRate }` is a shorthand for `{ tracingSamplingRate: tracingSamplingRate}`, so that means create an object with a key `tracingSamplingRate` for which the value is whatever is in the `tracingSamplingRate` variable.
So you're right, you can call the method with an object